apps/portal/examples/postgrest-template/content/source/Dockerfile.migrate

```
@@ -0,0 +1,28 @@
+# Dockerfile.migrate: Build a Docker image containing golang-migrate and SQL migration scripts
+# This image is used by ArgoCD PreSync hooks to run database migrations before deploying PostgREST
+
+FROM golang:1.22-alpine AS builder
+
+# Install wget for downloading golang-migrate
+RUN apk add --no-cache wget
+
+# Download golang-migrate binary
+RUN wget -qO- https://github.com/golang-migrate/migrate/releases/download/v4.17.0/migrate-linux-amd64.tar.gz | tar xz -C /tmp/
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)